timedctl/timedctl.py

```diff
@@ -400,19 +418,17 @@ def get_activities(self, date):
                 # Temporary timedelta
                 tmp_timedelta = to_time - attributes["from-time"]
                 # Add the rounded timedelta to the total time
-                rounding_factor = datetime.timedelta(minutes=15)
+                rounding_factor = timedelta(minutes=15)
                 rounded_time = (
-                    (tmp_timedelta + rounding_factor - datetime.timedelta(seconds=1))
+                    (tmp_timedelta + rounding_factor - timedelta(seconds=1))
                     // rounding_factor
                     * rounding_factor
                 )
                 total_time += rounded_time
 
-            table.append([activity_fmt, comment, from_time_fmt, to_time_fmt])
+            table.add_row(activity_fmt, comment, from_time_fmt, to_time_fmt)
 
-        output = terminaltables.SingleTable(table)
-        msg(f"Activities for {date if date is not None else 'today'}:")
-        click.echo(output.table)
+        print(table)
```

Comment:
Don't use print statements.

Reply:
its `rich.print` otherwise it just prints something like `<rich.Tablex23988219389xjfkdsaf>